packages/react/src/components/ComboBox/ComboBox.mdx

```diff
@@ -0,0 +1,30 @@
+import { Story, Props, Source, Preview } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
+import * as stories from './ComboBox-story.js';
+import ComboBox from '../ComboBox';
+
+# ComboBox
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component API](#component-api)
+- [Feedback](#feedback)
+
+## Overview
```

Comment:
Might be able to use the same strategy as TJ did over in Dropdown for use-cases / component API: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Dropdown/Dropdown.mdx

Some other examples could be:

- Skeleton
- Controlled vs uncontrolled
- Invalid states
- Label guidance (how to label it so that it's accessible)